CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+### 4.0.0
+
+* `relativePaths` is now automatically set to `false` when `publicPath` is used
+* `publicPath` now uses `webpack.config.js`'s `output.publicPath` by default (when not set in `OfflinePlugin` itself)
+* Added `AppCache.output` which is replacement for `AppCache.directory`. The last is deprecated now
+* Added `ServiceWorker.publicLocation` and `AppCache.publicLocation`
```

Comment:
I would more go for something like "outputPath" (something (the output) is created & put at a given path) but maybe I am missing something.